content/docs/tasks/telemetry/distributed-tracing/jaeger/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Jaeger
+description: How to configure the proxies to send tracing requests to Jaeger.
+weight: 10
+keywords: [telemetry,tracing,jaeger]
+---
+
+## Before you begin
+
+*   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
+
+    Either use the `istio-demo.yaml` or `istio-demo-auth.yaml` template, which includes tracing support, or
+    use the Helm chart with tracing enabled by setting the `--set tracing.enabled=true` option.
+
+* Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
```

Comment:
You link to the sample but not to the instructions on how to deploy it. I would be fine with just the link to the deployment instructions but ideally, you would link to both.